src/rules/store/prefix-selectors-with-select.ts

```diff
@@ -38,9 +46,52 @@ export default ESLintUtils.RuleCreator(docsUrl)<Options, MessageIds>({
             },
           },
           node: id,
-          messageId,
+          messageId: prefixSelectorsWithSelect,
+          suggest: [
+            {
+              messageId: prefixSelectorsWithSelectSuggest,
+              fix: (fixer) =>
+                fixer.replaceTextRange(
+                  [id.range[0], id.typeAnnotation?.range[0] ?? id.range[1]],
+                  suggestedName,
+                ),
+              data: {
+                name: suggestedName,
+              },
+            },
+          ],
         })
       },
     }
   },
 })
+
+function getSuggestedName(name: string) {
+  const selectWord = 'select'
+  // Ex: 'selectfeature' => 'selectFeature'
+  let possibleReplacedName = name.replace(
+    RegExp(`^${selectWord}(.+)`),
+    (_, lowercasedWord: string) => {
+      return `${selectWord}${capitalize(lowercasedWord)}`
+    },
+  )
+
+  if (name !== possibleReplacedName) {
+    return possibleReplacedName
+  }
+
+  // Ex: 'getCount' => 'selectCount'
+  possibleReplacedName = name.replace(
+    /[a-z\d]+([A-Z][A-Za-z\d]+)/,
```

Comment:
If I'm not mistaken, this would also replace other words as `get`, right?
e.g. `customersWithOrders` would become `selectWithOrders` .
If that's the case, I don't think it's a good idea and that we should just check for `get`

Reply:
Hmm yes, I intentionally did this. But after thinking more I'll go with your suggestion.